pkg/action/uninstall.go

```diff
@@ -122,7 +127,7 @@ func (u *Uninstall) Run(name string) (*release.UninstallReleaseResponse, error)
 	deletedResources, kept, errs := u.deleteRelease(rel)
 	if errs != nil {
 		u.cfg.Log("uninstall: Failed to delete release: %s", errs)
-		return nil, errors.Errorf("failed to delete release: %s", name)
+		return nil, uninstallError{error: errors.Errorf("failed to delete release: %s", name), uninstallErrors: errs}
```

Comment:
I wasn't 100% sure if changing the error message would be considered a breaking change which is why I left it like it was. If it isn't I think this should be updated to join the errors like the other places in this method do, that way all of the errors are surfaced to the CLI